bin/checkScheduledPush.ts

```diff
@@ -0,0 +1,9 @@
+import '../src/config';
+import { addMinutes } from 'date-fns';
+import { sendReadingReminderPush } from '../src/onesignal';
+
+(async (): Promise<void> => {
```

Comment:
This whole file needed?
Feels like a test for your own user?

Reply:
Thought it could be nice utility for anyone who wants. But can remove!